pkg/util/validation/limits.go

```diff
@@ -195,6 +198,8 @@ func (l *Limits) RegisterFlags(f *flag.FlagSet) {
 	f.Var(&l.ActiveSeriesCustomTrackersConfig, "ingester.active-series-custom-trackers", "Additional active series metrics, matching the provided matchers. Matchers should be in form <name>:<matcher>, like 'foobar:{foo=\"bar\"}'. Multiple matchers can be provided either providing the flag multiple times or providing multiple semicolon-separated values to a single flag.")
 	f.Var(&l.OutOfOrderTimeWindow, "ingester.out-of-order-time-window", "Non-zero value enables out-of-order support for most recent samples that are within the time window in relation to the TSDB's maximum time, i.e., within [db.maxTime-timeWindow, db.maxTime]). The ingester will need more memory as a factor of rate of out-of-order samples being ingested and the number of series that are getting out-of-order samples. A lower TTL of 10 minutes will be set for the query cache entries that overlap with this window.")
 
+	f.StringVar(&l.SeparateMetricsLabel, "validation.separate-metrics-label", "team", "Label used to further separate specific metrics")
```

Comment:
Nit: description should end in a dot.

Also, I feel that this description can be improved, but I don't have a specific suggestion now.

Reply:
"Label used to further separate specific metrics"

Couple of questions that might help clarify things:
* How will the label further separate specific metrics?
* What specific metrics will be separate?
* Separate from what?